src/data_loading.py

```diff
@@ -470,7 +515,10 @@ def create_clean_subjects(subjects_raw, screening_sites, display_terms_dict, dis
                 if subjects[i].dtype == np.float64:
                     # for display columns where data is numeric, merge on display dictionary, treating cols as floats to handle nas
                     display_terms[i] = display_terms[i].astype('float64')
-                subjects = subjects.merge(display_terms, how='left', on=i)
+                try:
+                    subjects = subjects.merge(display_terms, how='left', on=i)
+                except Exception as e:
+                    raise e
```

Comment:
```suggestion
                subjects = subjects.merge(display_terms, how='left', on=i)
```

If nothing is done when catching and then re-raisign exception, then maybe best to not put in try/except.  Same applies to line 554.

Reply:
I'll give it a try, this was originally put in to address an issue with the legacy code having a casting error.